include/upm_fti.h

```diff
@@ -30,6 +30,9 @@
 extern "C" {
 #endif
 
+/* Get filename w/o path */
+#define __FILENAME__ (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__)
```

Comment:
Should we also include string.h in this header file since we are using strrchr()?